test/Unit/App/Middleware/ContentNegotiationMiddlewareTest.php

```diff
@@ -101,9 +101,10 @@ public function testCannotResolveRepresentation(): void
 
     public function testFormatAcceptRequest(): void
     {
-        $this->assertIsArray(
-            $this->subject->formatAcceptRequest('application/json')
-        );
+        $accept = $this->subject->formatAcceptRequest('application/json');
+
+        $this->assertNotEmpty($accept);
```

Comment:
```suggestion
```
`assertNotEmpty` is useless because `assertSame` on the next line automatically implies that `$accept` is not empty.